### PR TITLE
Allow gke cluster to pull from gcr and remove kubernetes dashboard fr…

### PR DESCRIPTION
**Pull request by eduartua**
…om addons config

### CODE REVIEW

File: modules/gcp/gke/main.tf [hunk 1]
```diff
@@ -152,6 +149,14 @@ resource "google_project_iam_member" "service_account-roles" {
   member  = "serviceAccount:${google_service_account.gke_service_account.email}"
 }
```
Comment (victortrac) [inline modules/gcp/gke/main.tf]: Please put a comment here about why this is needed.

Comment (eduartua) [reply modules/gcp/gke/main.tf]: Done.

Comment (victortrac) [reply modules/gcp/gke/main.tf]: This comment doesn't match what you're actually doing. How does `roles/storage.objectViewer` provide access to stackdriver logging, trace, etc?  

The real issue is that `roles/storage.objectViewer` doesn't seem obvious that it grants access to pull from GCR. You managed to explain the real purpose for this role in the PR but yet you put something completely unrelated in the actual comment. This is worse than not having something at all.

Comment (eduartua) [reply modules/gcp/gke/main.tf]: The comment is not related to `roles/storage.objectViwer` but the previous resource declaration where `"roles/logging.logWriter"`, `"roles/monitoring.editor"`, `"roles/cloudtrace.agent"` are assigned to the gke SA. I'll add a comment to the resource `resource "google_storage_bucket_iam_binding" "gke_cluster_members"` that is the one you are talking about.

**Pull request merged**